### PR TITLE
Export to word and LaTeX, reference improvements and packaging

### PR DESCRIPTION
**Pull request by fwkoch**
This enables exporting a word document from a local md/ipynb file:
```
curvenote export word my-file.md
```

Previously, this command only worked with links to curvenote articles.

Support for the following features is tested:
- frontmatter
- citations (as long as `.bib` file is present in same directory as file)
- footnotes
- images

Citations are dependent on this update to `schema`: https://github.com/curvenote/schema/pull/74
- [ ] TODO: Bump the `schema` version once that's released ^

### CODE REVIEW

File: apps/cli/src/export/word/schema.ts [hunk 1]
```diff
@@ -152,20 +152,19 @@ export function getNodesAndMarks(styles?: Styles) {
...
       // TODO: This is probably very wrong
-      const id = oxa.id || oxa.block.block.slice(0, 8);
+      const id = oxa?.id || oxa?.block.block.slice(0, 8);
       switch (kind) {
         case ReferenceKind.cite:
           state.text(text);
           break;
         case ReferenceKind.table:
         case ReferenceKind.code:
         case ReferenceKind.fig:
```
Comment (fwkoch) [inline apps/cli/src/export/word/schema.ts:165]: I don't think fig/eqn references are going to work quite yet since they are using `oxa` ids still.

Comment (fwkoch) [reply apps/cli/src/export/word/schema.ts:165]: Ok - cross references don't work, but they just need support in schema: 
image

Other oxa id stuff is also not going to work, but that probably won't come up much in local md files.

File: packages/jtex/package.json [hunk 2]
```diff
@@ -32,17 +32,17 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/curvenote/citation-js-utils.git"
+    "url": "git+https://github.com/curvenote/curvenote.git"
   },
   "keywords": [
-    "citation-js"
+    "jtex"
```
Comment (rowanc1) [inline packages/jtex/package.json]: latex, jinja

File: apps/cli/src/store/local/intersphinx.ts [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+import zlib from 'zlib';
+import fetch from 'node-fetch';
+import { isUrl } from '../../utils';
+
+export type InventoryItem = { location: string; display?: string };
+export type InventoryData = Record<string, Record<string, InventoryItem>>;
+
+const DEFAULT_INV_NAME = 'objects.inv';
+
+export class Inventory {
```
Comment (stevejpurves) [inline apps/cli/src/store/local/intersphinx.ts]: @rowanc1 @fwkoch We're introducing intersphinx to manage a part of the export process?
curious to learn about what its role is and whether we are leaning on it for general cross referencing beyond latex/doc export?

Comment (rowanc1) [reply apps/cli/src/store/local/intersphinx.ts]: This is certainly wider scope than the export to latex. It has come in here as I have built up the docs, and some of the conversations in the myst ecosystem on referencing and links.

I am also planning to change the links to follow the updates in that world.

See the general channel of EBP slack for some more context on this.

**Comment by rowanc1** [comment]
@fwkoch not sure why this one cannot merge. I have put up two other PRs that I think are the same?

They are pointing to this branch currently, but they can both merge to main without conflict. If one of them looks right we can take it forward and merge this afternoon.

**Comment by rowanc1** [comment]
Merged in #322.

**Pull request closed without merging**